app/controllers/users/sessions_controller.rb

```diff
@@ -9,6 +9,8 @@ def after_sign_in_path_for(resource)
         resource.display_whats_new = false
         resource.save!
         static_path('whats-new')
+      elsif resource.training_emails.nil?
```

Comment:
Is it possible for someone to skip the second field now? I notice you changed to only check one.

Reply:
Updated [088d9a3](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/659/commits/088d9a3236a1d392aaef71939822b94962408d63)